web/src/OverviewLogPane.tsx

```diff
@@ -0,0 +1,282 @@
+import Anser from "anser"
+import React, { Component } from "react"
+import styled, { keyframes } from "styled-components"
+import "./LogPane.scss"
+import "./LogPaneLine.scss"
+import LogStore, { useLogStore } from "./LogStore"
+import PathBuilder, { usePathBuilder } from "./PathBuilder"
+import { SizeUnit } from "./style-helpers"
+import { LogLine } from "./types"
+
+type OverviewLogComponentProps = {
+  manifestName: string
+  pathBuilder: PathBuilder
+  logStore: LogStore
+}
+
+let LogPaneRoot = styled.section`
+  padding-top: ${SizeUnit(0.5)};
+  padding-bottom: ${SizeUnit(0.5)};
+  width: 100%;
+  height: 100%;
+  overflow-y: auto;
+  box-sizing: border-box;
+`
+
+const blink = keyframes`
+0% {
+  opacity: 1;
+}
+50% {
+  opacity: 0;
+}
+100% {
+  opacity: 1;
+}
+`
+
+let LogEnd = styled.div`
+  animation: ${blink} 1s infinite;
+  animation-timing-function: ease;
+  padding-topt: ${SizeUnit(0.25)};
+  padding-left: ${SizeUnit(0.625)};
+`
+
+let anser = new Anser()
+
+function newLineEl(line: LogLine, isContextChange: boolean): Element {
+  let text = line.text
+  let level = line.level
+  let buildEvent = line.buildEvent
+  let classes = ["LogPaneLine"]
+  if (level === "WARN") {
+    classes.push("is-warning")
+  } else if (level === "ERROR") {
+    classes.push("is-error")
+  }
+  if (isContextChange) {
+    classes.push("is-contextChange")
+  }
+  if (buildEvent === "init") {
+    classes.push("is-buildEvent-init")
+  }
+  if (buildEvent === "fallback") {
+    classes.push("is-buildEvent-fallback")
+  }
+  let span = document.createElement("span")
+  span.classList.add(...classes)
+  let code = document.createElement("code")
+  code.classList.add("LogPaneLine-content")
+  code.innerHTML = anser.linkify(
```

Comment:
OMG at long last! No longer shall we hound our users with links they have to copy-paste... 👏🏼